integreat_compass/cms/views/index.py

```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import render
+
+
+def get(request):
+    """
+    Start page of the project
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :return: The rendered template response
+    :rtype: ~django.template.response.TemplateResponse
+    """
+    return render(request, "index.html")
```

Comment:
```suggestion
from django.views.generic import TemplateView


class IndexView(TemplateView):
    """
    Start page of the project
    """

    template_name = "index.html"
```
I know it's just a placeholder, but I think it would be cool to directly use Django's class-based views here to encourage the usage of class-based views in general when more views are added :sweat_smile: